applications/sandbox/assets/shaders/flipbook.shs

```diff
@@ -0,0 +1,25 @@
+#version 450
+#state CULL OFF
+generate(vertex, fragment)
+
+#include <stdio.shinc>
+
+
+shader(vertex)
+{
+    void main(void)
+    {
+        gl_Position = ModelToScreenSpacePosition(position);
+        PropagateStdIO();
+    }
+}
+
+shader(fragment)
+{
+    uniform sampler2DArray textures;
+    void main(void)
+    {
+        //SubmitOutput(mix(texture(textures, vec3(stdio(uv),0)),texture(textures, vec3(stdio(uv),1)),0.5));
+        SubmitOutput(texture(textures, vec3(stdio(uv),0)));
```

Comment:
i'd say at least remove the commented out line and make this actually use the frameId